jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/RawConfiguration.java

```diff
@@ -57,6 +57,12 @@
     List<String> getExcludesList();
   }
 
+  interface CredHelperConfiguration {
```

Comment:
```suggestion
  static interface CredHelperConfiguration {
```

Reply:
I believe the `CredHelperConfiguration` interface is implicitly static - why does it need to be explicitly declared as static?

Reply:
Ah, you're totally right. If you want, you're welcome to delete unnecessary `static` in other nested interfaces.